scripts/ci-frontend-metrics.sh

```diff
@@ -10,7 +10,7 @@ CLASSNAME_PROP="$(grep -r -o -E --include="*.ts*" "\.*.className=\W.*\W.*" publi
 EMOTION_IMPORTS="$(grep -r -o -E --include="*.ts*" --exclude="*.test*" "\{.*css.*\} from '@emotion/css'" public/app | wc -l)"
 TS_FILES="$(find public/app -type f -name "*.ts*" -not -name "*.test*" | wc -l)"
```

Comment:
there's a lot more occurrences (e.g. here) where we assume there's a `public` folder instead of `/grafana/public` - do these need to be updated as well?

Reply:
I think `public/app` might want to stay the same but if there any other occurrences  of compiled typescript files we'll need to update those.